sdk/servicebus/service-bus/test/serviceBusClient.spec.ts

```diff
@@ -1218,60 +1029,6 @@ describe("Errors after close()", function(): void {
   describe("Errors after close() on sender", function(): void {
     const entityToClose = "sender";
 
-    it("Partitioned Queue: errors after close() on sender", async function(): Promise<void> {
-      await beforeEachTest(
-        TestClientType.PartitionedQueue,
-        TestClientType.PartitionedQueue,
-        entityToClose
-      );
-
-      await testSender(
-        getSenderClosedErrorMsg(senderClient.entityPath, ClientType.QueueClient, false)
-      );
-    });
-
-    it("Partitioned Queue with sessions: errors after close() on sender", async function(): Promise<
-      void
-    > {
-      await beforeEachTest(
-        TestClientType.PartitionedQueueWithSessions,
-        TestClientType.PartitionedQueueWithSessions,
-        entityToClose,
-        true
-      );
-
-      await testSender(
-        getSenderClosedErrorMsg(senderClient.entityPath, ClientType.QueueClient, false)
-      );
-    });
-
-    it("Partitioned Topic: errors after close() on sender", async function(): Promise<void> {
-      await beforeEachTest(
-        TestClientType.PartitionedTopic,
-        TestClientType.PartitionedSubscription,
-        entityToClose
-      );
-
-      await testSender(
-        getSenderClosedErrorMsg(senderClient.entityPath, ClientType.TopicClient, false)
-      );
-    });
-
-    it("Partitioned Topic with sessions: errors after close() on sender", async function(): Promise<
-      void
-    > {
-      await beforeEachTest(
-        TestClientType.PartitionedTopicWithSessions,
-        TestClientType.PartitionedSubscriptionWithSessions,
-        entityToClose,
-        true
-      );
-
-      await testSender(
-        getSenderClosedErrorMsg(senderClient.entityPath, ClientType.TopicClient, false)
-      );
-    });
-
     it("Unpartitioned Queue: errors after close() on sender #RunInBrowser", async function(): Promise<
```

Comment:
Since it is the sender that is being tested here, we do not need the variations between sessions and no sessions.

Reply:
Since close() is not a pure in memory operation and involves an AMQP operation, wouldn't this add coverage for the interaction with service side implementation?

Reply:
The code in close() as far as I know is purely client side code, whether it is about the library setting appropriate flags, using the flag to throw errors or cleaning up the amqp links in rhea.
None of these differ on what kind of entity is being used. Therefore, picking any of the entities whether it has sessions enabled or not, doesnt make a difference

Reply:
Updated. 
But since we also have a service side implementation of the AMQP / rhea interaction side of things. (Where events and stuff get raised). Seems useful generally to have coverage for tests that rely on those events to get raised (and to ensure they get raised inspite of the variations in the request flags).
Though agree that it may be possible that close() may not be taking in anything more than an identifier. Hopefully the implementation on service side is genric enough too, but seems unlikely because I think partitioned vs unpartitioned entities hitting separate systems/endpoins iirc